validation/markets-validator.js

```diff
@@ -52,7 +52,7 @@ var MIN_LONGITUDE = -180.0;
  * is impossible for us to resolve the issue.
  * The aim is to keep this count as low as possible.
  */
-var ACCEPTABLE_WARNINGS_COUNT = 4; // Rostock, Stralsund, Bremen, Chemnitz user agent
+var ACCEPTABLE_WARNINGS_COUNT = 7; // Bremen, Chemnitz, Frankfurt am Main, Münster, Solingen, Trier, Zwickau
```

Comment:
What happend to Rostock?

Reply:
These cities respond very unreliable. I almost cannot run any job more than once. Increasing the count is a temporary solution. One needs to see if the validation script can be improved. I personally do not have time currently to do so.